lighthouse-plugin-ad-speed-insights/gatherers/iframe-elements.js

```diff
@@ -19,6 +19,43 @@ const pageFunctions = require('lighthouse/lighthouse-core/lib/page-functions.js'
 
 /* eslint-env browser, node */
 
+/**
+ * TODO(jburger): Move to page-functions.js once integrated with Lighthouse.
+ * @param {?HTMLElement} element
+ * @return {Boolean}
+ */
+function isFixed(element) {
+  // @ts-ignore
+  const scrollableAncestor = ((element) => {
+    let currentEl = element.parentElement;
+    while (currentEl) {
+      if (currentEl.scrollHeight > currentEl.clientHeight) {
+        // Check style before computedStyle as computedStyle is expensive.
+        const yScroll = currentEl.style.overflowY ||
+          window.getComputedStyle(currentEl).overflowY;
```

Comment:
How about moving `el.style.foo || window.getComputedStyle(currentEl).foo` to a helper function?